config/secrets.yml

```diff
@@ -39,6 +39,7 @@ staging:
   notify_custodian_invite_template: <%= ENV["NOTIFY_CUSTODIAN_INVITE_TEMPLATE"] %>
   notify_advanced_invite_template: <%= ENV["NOTIFY_ADVANCED_INVITE_TEMPLATE"] %>
   notify_basic_invite_template: <%= ENV["NOTIFY_BASIC_INVITE_TEMPLATE"] %>
+  google_analytics_tracking_id: <%= ENV['UA_TRACKING_ID'] %>
```

Comment:
as this isn't a secret can we just set it in the normal config file e.g. `staging.rb`?

Reply:
Rails convention is `ENV['foo']`, secrets was introduced in Rails 4.1 to keep "things" in one place